tests/test_taxi.py

```diff
@@ -69,7 +67,7 @@ def test_car_is_register_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
The assertion `assert file_exists` should be replaced with `self.assertTrue(file_exists)` to maintain consistency with Django's `TestCase` methods and provide better error messages on failure.